src/server/pfs/server/testing/server_test.go

```diff
@@ -341,6 +341,7 @@ func TestPFS(suite *testing.T) {
 
 		c := env.PachClient
 		badFormatErr := "only alphanumeric characters"
+		uniqueConstraintErr := "duplicate key value violates unique constraint \"projects_name_key\" (SQLSTATE 23505)"
```

Comment:
`uniqueConstraintErr` is only used once, so no need to instantiate a variable here.